src/hotspot/share/opto/loopTransform.cpp

```diff
@@ -1968,6 +1975,34 @@ void PhaseIdealLoop::update_main_loop_skeleton_predicates(Node* ctrl, CountedLoo
   }
 }
 
+void PhaseIdealLoop::insert_post_loop_skeleton_predicates(LoopNode* main_loop_head, CountedLoopNode* post_loop_head, Node* init, Node* stride) {
```

Comment:
Maybe this could be renamed to `copy_skeleton_predicates_to_post_loop()` to be consistent with the method `copy_skeleton_predicates_to_main_loop()`?